zuul.d/projects.yaml

```diff
@@ -118,15 +118,14 @@
         - cluster-api-provider-openstack-image-build:
             branches: master
 
-#NOTES: Should be updated when switch to official project
 - project:
-    name: zhongjun2/bosh-huaweicloud-cpi-release
+    name: cloudfoundry-incubator/bosh-huaweicloud-cpi-release
     check:
       jobs:
-        - bosh-huaweicloud-cpi-release-acceptance-test
+        - bosh-huaweicloud-cpi-release-acceptance-test-huaweicloud
```

Comment:
should be in `check-generic-cloud` pipeline

Reply:
oh... seems I have override my previous change :(

Reply:
no problem